src/protocol/data/memcache/response.c

```diff
@@ -18,6 +18,8 @@ struct bstring rsp_strings[] = {
 FREEPOOL(rsp_pool, rspq, response);
 static struct rsp_pool rspp;
 static bool rspp_init = false;
+/* TODO(simms): this should probably be readable from rust */
+static size_t rsp_val_buf_size = RSP_VAL_BUF_SIZE;
```

Comment:
I wasn't sure how to make this visible in rust. I guess having a function rust can call to get the value would be one way.

Reply:
in a follow-up branch i've implemented this as a param that gets passed through to the cdb handle creation method.